oap-server/server-cluster-plugin/cluster-consul-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/consul/ClusterModuleConsulProvider.java

```diff
@@ -76,6 +76,10 @@ public ClusterModuleConsulProvider() {
 //                    we should set this value or it will be blocked forever
                     .withConnectTimeoutMillis(3000);
 
+            if (config.getAclToken() != null && !config.getAclToken().isEmpty()) {
```

Comment:
Why do not you use `!Strings.isNullOrEmpty(config.getAclToken())`? I think that it is more readable.

Reply:
@web-xiaxia this should be fixed

Reply:
Replacement completed